### DISCUSSION TITLE
[GitHub] Issue with visibility of accounts

### DISCUSSION THREAD

**Discussion by mdziczkowski** [community/community#43130] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I had  found a following issue regarding visibility of accounts on GitHub:

I wanted to add an friend to one of my repisitories, but after typing his name in the "Invite" page, I dot an error that he isn't an member of GitHub, even if I can normally view his profile

**Comment by jfullstackdev** [comment]
can you share the exact user ? so we can check it too
